### DISCUSSION TITLE
Authorization for API

### DISCUSSION THREAD

**Discussion by ChennaiTechie** [ragapp/ragapp#220] [Q&A]
**Is your feature request related to a problem? Please describe.**
API Endpoint of Ragapp is reachable without authentication.

**Describe the solution you'd like**
Enabling Token based authentication for the API endpoints will solve the issue

**Comment by leehuwuj** [comment]
Hi @ChennaiTechie 
We provided an example deployment with Keycloak and Traefik for both authentication and authorization. Please check it out at: https://github.com/ragapp/ragapp/tree/main/deployments/multiple-ragapps
